spec/features/when_nomis_is_missing_information_spec.rb

```diff
@@ -42,7 +42,7 @@
           stub_bookings = [{
             bookingId: 1,
             sentenceDetail: {
-              releaseDate: 30.years.from_now.iso8601,
+            releaseDate: 30.years.from_now.iso8601,
```

Comment:
this is no longer valid - hence the (unintended) change below. I think this needs to be conditionalReleaseDate as above?

Reply:
This was to show that we do not have any of the required dates the page would not crash, admittedly I could have just removed the 'releaseDate' and the result would have been the same. It would probably be clearer to the reader. I'll fix that